packages/hardhat/src/index.ts

```diff
@@ -1,2 +1,2 @@
-import "../../hardhat-forge/src/index";
-import "../../hardhat-anvil/src/index";
+import "@foundry-rs/hardhat-forge";
```

Comment:
Does this actually work? It looks like its just importing and not exporting.

Google search says you can do something like:
```ts
export * from "@foundry-rs/hardhat-forge";
```

Reply:
Yeah if you want to export, you'd want that. Can also do `export * as forge from "@foundry-rs/hardhat-forge";` if you want the exports to have a name vs import each thing from it.

the compiled version currently is the equivalent of just the imports, so

```js
require("@foundry-rs/hardhat-forge");
require("@foundry-rs/hardhat-anvil");
```

Reply:
nice, using aliases now